pages/about.html

```diff
@@ -0,0 +1,307 @@
+              About
+            </a>
+          </li>
+        </ul>
+      </div>
+      <div class="title-container">
+        <p class="hello">"World-Health-Organization"</p>
+        <h1 class="title">Seventy-fifth World Health <br> Assembly  <br>
+          2022</h1>
+      </div>
+
+      <p class="supporting-text">The World Health Assembly is the decision-making body of WHO. It is attended by delegations from all WHO Member States and focuses on a specific health agenda prepared by the Executive Board.</p>
+      <div class="contact">
+        <p>Please contact us per Email for any further
+          questions about WHO 2022!</p>
+        <h3>alusinejalloh800@gmail.com</h3>
+      </div>
+
+    </section>
+
+    <section class="about-logo-section">
+      <div class="title-group">
+        <h2 class="title">WORLD-HEALTH-ORGANIZATION 2022 Logo
+        </h2>
+        <hr>
+      </div>
+
+      <p class="supporting-text">
+        Seventy-fifth World Health Assembly. You can refer contact @
+        alusinejalloh800@gmail.com
+      </p>
+
+      <div class="logo-container">
+        <div class="logo">
+          <img src="../images/h-logo-white.svg" alt="logo">
+           
+        </div>
+        <small>2022.08.09 (MON)~ 04 (FRI)</small>
+      </div>
+
+
+    </section>
+
+    <section class="past-events-section">
+      <div class="title-group">
+        <h2 class="title">See past WHO events</h2>
+        <hr>
+        <p class="supporting-text">
+          The two most famous World Health Organization Assembly
+          tournaments in history
+        </p>
+      </div>
+      <ul class="event-list">
+        <li class="event event1">
+          <div class="event-info">
+            <h2 class="date"> 18-22 April 1977</h2>
+            <p>  30th World Health Assembly, Geneva</p>
+          </div>
+        </li>
+
+        <li class="event event2">
+          <div class="event-info">
+            <h2 class="date">20–22 May 2019</h2>
+            <p>72nd Word Health Assembly in Geneva</p>
+          </div>
+        </li>
+
+      </ul>
+    </section>
+
+    <section class="partners-section">
+      <div class="title-group">
+        <h2 class="title">Partner</h2>
+        <hr>
+      </div>
+      <ul>
+        <li>
+          
+          HEPA Europe
+        </li>
+        <li>
+          
+          ASEAN
+        </li>
+        <li>
+          
+         NITAGs
+        </li>
+        <li>
+         SEEHN
+        </li>
+        <li>
+         ETAGE
+        </li>
+      </ul>
+    </section>
+  </main>
+
+  <footer class="license">
+    
+    <a class="logo" href="#">
+      <img src="../images/h-logo-white.svg" alt="logo">
+       
+    </a>
+    <p class="logo-text">
+      2022 World health Assembly. All Rights Reserved.
+      This is registred under the MIT license. for more
+      info, please <a
+        href="https://github.com/Alusp/World-Health-Assembly">See
+        here</a>
+    </p>
+
+  </footer>
```

Comment:
- [ ] Your footer should have the logo aligned to the left and the text to the right both on desktop and mobile versions as it is in the template.
- [ ] It's really nice that you tried to follow the World Health Organization colors from their website but it makes this project color theme a little bit awkward. The provided template has a white background with dark text and logo for both pages on desktop and mobile versions and a dark background with white text for the About page desktop view. Kindly try to follow these rules on your project.
Template:
<img width="748" alt="image" src="https://user-images.githubusercontent.com/61919617/181599222-caa843b8-b853-467a-b707-2190e1e80efe.png">
<img width="405" alt="image" src="https://user-images.githubusercontent.com/61919617/181599289-b290313e-9c6e-44b6-a7b4-40564546dbf3.png">